src/components/AddMovieModal/__snapshots__/AddMovieModal.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AddMovieModal component should render AddMovieModal snapshot 1`] = `<DocumentFragment />`;
```

Comment:
There should be a markup for **AddMovieModal** 
Because in this implementation test will pass even you change **AddMovieModal**  component